setup.cfg

```diff
@@ -34,6 +34,7 @@ dev =
     flake8
     isort
     mypy
+    numpy
```

Comment:
It might be better to install this via `pytest-regressions[image]` instead of installing directly.

Reply:
Nice catch